bigquery/google/cloud/bigquery/job.py

```diff
@@ -90,46 +90,95 @@ def _error_result_to_exception(error_result):
 
 
 class Compression(_EnumApiResourceProperty):
-    """Pseudo-enum for ``compression`` properties."""
+    """The compression type to use for exported files.
+
+    Possible values include `GZIP` and `NONE`. The default value is `NONE`.
+    """
     GZIP = 'GZIP'
     NONE = 'NONE'
 
 
 class CreateDisposition(_EnumApiResourceProperty):
-    """Pseudo-enum for ``create_disposition`` properties."""
+    """Pseudo-enum for ``create_disposition`` properties.
```

Comment:
For consistency with your other changes, please rewrite the summary line to remove the "Pseudo-enum" bit.